content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,239 @@
+
+- Run large-scale or resource-intensive batch changes without clogging your local machine
+- Run large batch changes quickly by distributing them across an autoscaled pool of compute instances
+- Get a better debugging experience with the streaming of logs directly into Sourcegraph.
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [Code Intelligence auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs) as well. One or more executors are available by default on Cloud instances of Sourcegraph.
+
+[Batch Changes]
+
+[Docs](https://docs.sourcegraph.com/batch_changes/explanations/server_side)
+
+- New packaging updates
+
+##### ☁️ Enterprise Cloud dedicated(/managed?) instances
+
+- [Secure, easy, and scalable Sourcegraph Cloud dedicated(/managed?) instances for the enterprise](/blog/enterprise-cloud)
+
+Headline: [Do X] with [Z]
+Description: 2 - 4 sentences on the feature, functionality and benefits
+Media: provide a screenshot, gif, or video to help support the copy
+Tag by product: [Code Search], [Batch Changes], [Code Navigation], [Code Insights], [Admin]
+Docs: provide a link to the updated docs
+
+##### 💖 For admins
+
+### Get a deep understanding of usage and measure the value of our platform with new and enhanced in-product analytics
+With 3.42, we introduced an all-new analytics including [XYZ]. Our new analytics make it easier for admins to understand user engagement, measure efficiency in terms of time saved, and help calculate the value of utilizing Sourcegraph and its features. We’ve significantly improved our in-product analytics to include the projected value and dev-time-saved for all Sourcegraph features, the ability to export data, individual user usage table, and much more. 
+
+Media: provide a screenshot, gif, or video to help support the copy
+
+[Admin]
+
+Docs: provide a link to the updated docs
+
+- 1-click log export - at risk for 4.0 relase
+
+### Easily export traces using OpenTelemetry
+
+[OpenTelemetry (OTEL)](https://opentelemetry.io/) is an open specification for exporting observability data into a user’s preferred systems. We aim to make OpenTelmetry the default way to quickly and easily export observability data from Sourcgraph, starting with tracing in Sourcegraph 4.0. This is a breaking change since we now require customers to export tracing using the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/), which will now be deployed by default in all Sourcegraph deployment methods, replacing the Jaeger agents and collectors we used to ship by default.
+[Admin]
+Docs: provide a link to the updated docs
+
+### Quickly and easily get the status on your repository and sync permissions
```

Comment:
@ryphil looking for some help filling out the details here on what has been enhanced and the benefits.